eng/Versions.props

```
@@ -7,7 +7,7 @@
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
     <SdkBandVersion>8.0.100</SdkBandVersion>
-    <PackageVersionNet7>7.0.12</PackageVersionNet7>
+    <PackageVersionNet7>7.0.$([MSBuild]::Add($(PatchVersion),14))</PackageVersionNet7>
```

If we do not ship 6 and 7 along side 8 in the next release this will need to change
```suggestion
    <PackageVersionNet7>7.0.$([MSBuild]::Add($(PatchVersion),13))</PackageVersionNet7>
```